test/e2e/kubectl.go

```diff
@@ -312,22 +313,21 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 
 			// Build the static kubectl
 			By("Finding a static kubectl for upload")
-			testStaticKubectlPath, err := findBinary("kubectl", "linux/amd64")
+			platform := runtime.GOOS + "/" + runtime.GOARCH
```

Comment:
Ah, missed that the machine that built the binary (the machine running the test), and the machine that's using the binary (a node in the cluster) aren't the same machines. 

Thought the `kubectl` binary would be used on the machine that's running the test
Fixing...